opendbc/can/dbc.cc

```diff
@@ -14,11 +14,11 @@
 #include "opendbc/can/common.h"
 #include "opendbc/can/common_dbc.h"
 
-std::regex bo_regexp(R"(^BO_ (\w+) (\w+) *: (\w+) (\w+))");
-std::regex sg_regexp(R"(^SG_ (\w+) : (\d+)\|(\d+)@(\d+)([\+|\-]) \(([0-9.+\-eE]+),([0-9.+\-eE]+)\) \[([0-9.+\-eE]+)\|([0-9.+\-eE]+)\] \"(.*)\" (.*))");
-std::regex sgm_regexp(R"(^SG_ (\w+) (\w+) *: (\d+)\|(\d+)@(\d+)([\+|\-]) \(([0-9.+\-eE]+),([0-9.+\-eE]+)\) \[([0-9.+\-eE]+)\|([0-9.+\-eE]+)\] \"(.*)\" (.*))");
-std::regex val_regexp(R"(VAL_ (\w+) (\w+) (\s*[-+]?[0-9]+\s+\".+?\"[^;]*))");
-std::regex val_split_regexp{R"([\"]+)"};  // split on "
+RE2 bo_regexp(R"(^BO_ (\w+) (\w+) *: (\w+) (\w+))");
+RE2 sg_regexp(R"(^SG_ (\w+) : (\d+)\|(\d+)@(\d+)([\+|\-]) \(([0-9.+\-eE]+),([0-9.+\-eE]+)\) \[([0-9.+\-eE]+)\|([0-9.+\-eE]+)\] \"(.*)\" (.*))");
+RE2 sgm_regexp(R"(^SG_ (\w+) (\w+) *: (\d+)\|(\d+)@(\d+)([\+|\-]) \(([0-9.+\-eE]+),([0-9.+\-eE]+)\) \[([0-9.+\-eE]+)\|([0-9.+\-eE]+)\] \"(.*)\" (.*))");
+RE2 val_regexp(R"(VAL_ (\w+) (\w+) (.*))");
+RE2 val_split_regexp(R"((([0-9]) \"(.+?)\"))");
```

Comment:
Notes:
- no tests match `\s*[-+]?` so this was removed
- RE2 doesn't support a variable number of matching groups.